js/defaults.js

```diff
@@ -8,7 +8,7 @@
  */
 
 var defaults = {
-	port: 8080,
+	port: process.env.MM_PORT || 8080,
```

Comment:
_Various whitespace characters can be inputted by programmers by mistake for example from copying or keyboard shortcuts. Pressing Alt + Space on OS X adds in a non breaking space character for example._
http://eslint.org/docs/rules/no-irregular-whitespace